src/openms/source/CHEMISTRY/EnzymaticDigestion.cpp

```diff
@@ -165,6 +166,10 @@ namespace OpenMS
       { // '0' is included in cleavage_positions, so starting fragments will be found as well
         spec_n = true;
       }
+      else if (allow_random_asp_pro_cleavage && (sequence[pos-1] == 'D') && (sequence[pos] == 'P'))
```

Comment:
can pos be 0?

Reply:
Yes but then it would be found in the first "if case" since 0 is always included in the cleavage_positions. I hope this assumption is not too loose.